sw/device/lib/testing/usb_testutils_diags.h

```diff
@@ -22,6 +25,13 @@
 // when simulating with the USB DPI module.
 #define USBUTILS_ENABLE_TRC 0
 
+// Prompt the user for intervention if we're not running in simulation;
+// logging within simulation can be time-consuming and induce failures.
+#define USBUTILS_USER_PROMPT(...)                                          \
+  if (kDeviceType != kDeviceSimDV && kDeviceType != kDeviceSimVerilator) { \
+    LOG_INFO(__VA_ARGS__);                                                 \
+  }
```

Comment:
To follow the `function-like macro` [style](https://opentitan.org/book/doc/contributing/style_guides/c_cpp_coding_style.html#constants-and-preprocessor-macros) 
```suggestion
#define USBUTILS_USER_PROMPT(...)                                          \
  do { \
    if (kDeviceType != kDeviceSimDV && kDeviceType != kDeviceSimVerilator) { \
    LOG_INFO(__VA_ARGS__);                                                 \
  }\
while(0)
```

Reply:
Thanks. I thought there must be a convention.